osde2e/osde2e-wrapper.py

```diff
@@ -371,6 +395,16 @@ def main():
         cluster_name_seed = cluster_name_seed[:6]
     cluster_name_seed += "-" + random_string
 
+    try:
+        logging.debug('Saving cluster name seed %s to the working directory' % cluster_name_seed)
+        seed_file = open(my_path + '/cluster_name_seed','x')
```

Comment:
Is it intended to fail if the file already exists?

Reply:
it will only run one time, so file will never exists, try is about having permissions to write on the folder